README.md

```diff
@@ -45,6 +45,8 @@ Please also check the documentation of [Language Support for Java by Red Hat](ht
 - `args` - The command line arguments passed to the program.
 - `sourcePaths` - The extra source directories of the program. The debugger looks for source code from project settings by default. This option allows the debugger to look for source code in extra directories.
 - `classPaths` - The classpaths for launching the JVM. If not specified, the debugger will automatically resolve from current project.
+- `classPathsToAdd` - Additional classpaths to add before launching the JVM.
+- `classPathsToRemove` - Classpaths to remove before when launching the JVM.
```

Comment:
A little concern about this option. Can you list some scenarios that needs this option?